torch/onnx/_internal/jit_utils.py

```diff
@@ -321,3 +321,13 @@ def get_device_from_value(value: _C.Value) -> Optional[torch.device]:
         return None
     tensor_type = typing.cast(_C.TensorType, value.type())
     return tensor_type.device()
+
+
+@_beartype.beartype
+def _parse_node_kind(kind: str) -> Tuple[str, str]:
+    if "::" not in kind:
+        raise torch.onnx.errors.OnnxExporterError(
```

Comment:
There are some discussions on what errors we should raise in these cases. I am leaning toward `ValueError` because that is more clear semantically. That's a discussion to be had. 

For you could you add a TODO and point it to [#76155](https://github.com/pytorch/pytorch/issues/76155)? Thanks!

Reply:
Given this is an internal tool, I think ValueError seems more reasonable. I kind of feel like OnnxExporterError is for user...